package.json

```diff
@@ -36,12 +36,12 @@
         "doc": "npm run doc:generate && npm run doc:deploy",
         "doc:generate": "rimraf ./ref/* && typedoc",
         "doc:deploy": "gh-pages -d ref -a -e ref",
-        "beachball:check": "beachball check --branch origin/dev",
-        "beachball:change": "beachball change --branch origin/dev",
-        "beachball:bump": "beachball bump --branch origin/dev --bumpDeps false && node release-scripts/updateVersion.js",
-        "beachball:bump:alpha": "beachball bump --branch origin/dev --bumpDeps false --prerelease-prefix alpha",
-        "beachball:bump:beta": "beachball bump --branch origin/dev --bumpDeps false --prerelease-prefix beta",
-        "beachball:bump:official": "beachball bump --branch origin/dev",
+        "beachball:check": "beachball check --branch origin/msal-v5",
+        "beachball:change": "beachball change --branch origin/msal-v5",
+        "beachball:bump": "beachball bump --branch origin/msal-v5 --bumpDeps false && node release-scripts/updateVersion.js",
```

Comment:
Recommend either removing this script or setting the prerelease-prefix so there's no chance of accidentally releasing v5 mainline before we're ready

Reply:
Good point, removed.